### DISCUSSION TITLE
Add support for Java Development Kit (JDK) 21

### DISCUSSION THREAD

**Discussion by nwcm** [forcedotcom/salesforcedx-vscode#5372] [Ideas]
The JDK 21 is the latest LTS as of September 2023. JDK 17 is still supported until September 2026 with extension to September 2029.

https://www.oracle.com/au/java/technologies/java-se-support-roadmap.html

**Comment by daphne-sfdc** [comment]
Hi @nwcm,
Thank you for this feature request.  Our team is aware of the rollout of JDK 21 and do plan to implement support for it in the future.  We'll let you know when we do.

**Comment by diyer** [comment]
Closing this one as this is not an issue.
